LibraBFT/Yasm/System.agda

```diff
@@ -40,9 +41,27 @@ module LibraBFT.Yasm.System
  SenderMsgPair : Set
  SenderMsgPair = PeerId × Msg
 
+ actionToSMP : PeerId → LYT.Action Msg → Maybe SenderMsgPair
+ actionToSMP pid (LYT.send m) = just (pid , m)
+
  SentMessages : Set
  SentMessages = List SenderMsgPair
 
+ -- Convert the actions a peer `pid` took to a list of `SentMessages`.
```

Comment:
It's not a list of `SentMessages` (`SentMessages` is itself a list).  Maybe "a list of sent messages" :smile:.